### PR TITLE
Add test case for JSON resource marshaling

### PR DESCRIPTION
**Pull request by vaibhavgholap23**
Hi creators,
I have added test cases for :
RepoMergeUpstreamResult struct
under Helps: #55 issue

I have followed all the guidelines to contribute to this repository.
Please revert in case of any changes are needed.
I am a first-time contributor to Hackoctober, It will be great if you review & accept my contribution.

### CODE REVIEW

File: github/repos_merging_test.go [hunk 1]
```diff
@@ -126,3 +126,21 @@ func TestRepositoriesService_MergeUpstream(t *testing.T) {
...
+	u := &RepoMergeUpstreamResult{
+		Message:    String("message"),
+		MergeType:  String("merge_type"),
+		BaseBranch: String("base_branch"),
+	}
+
+	want := `{
+		"message": "message",
+		"merge_type":"merge_type",
+		"base_branch":"base_branch"
```
Comment (gmlewis) [inline github/repos_merging_test.go:142]:
```suggestion
		"merge_type": "merge_type",
		"base_branch": "base_branch"
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Let's please format the `want` string with included whitespace as if `json.MarshalIndent` was the function that created it, for consistency across the repo.

File: github/repos_merging_test.go (hunk 1 above)
Comment (vaibhavgholap23) [reply github/repos_merging_test.go]: changes pushed

**Review by gmlewis — APPROVED** [review]
Thank you, @vaibhavgholap23 !
LGTM.
Merging.

**Pull request merged**